index.html

```diff
@@ -893,9 +893,12 @@
 This is a special case of the more general principle that data should not be used for more
 [=purposes=] than the data's subjects understood it was being collected for.
 
-A service should explain how it uses people's data to protect them and other people, and
-how it might additionally use someone's data if it believes that person has broken the
-rules.
+Services sometimes use people's data in order to protect those or other people.
+This is legitimate as long as the extra risk created by the data collection is
+less than the protection it offers and the extra risk isn't concentrated on
+[=vulnerable people=]. A service that does this should explain what data it's
```

Comment:
I'm not a fan of the vulnerable people language. I don't see how a service can support incompatible classes of vulnerable people, such as sex workers and children, with this language. A platform needs the freedom and liberty to make choices about which users they will preference, and that will inevitable disadvantage other users.